src/components/Text/types.ts

```diff
@@ -24,6 +24,21 @@ export enum TextVariants {
   snwHeaderLink = 'snwHeaderLink',
   snwHeadingHero1 = 'snwHeadingHero1',
   snwHeadingHero2 = 'snwHeadingHero2',
+
+  // New variants
+  display4XL = 'display4XL',
```

Comment:
can we add a eslint warning for all components that are using the old TextVariant, and old tokens in scss to warn that they're deprecated, and then bring this same lint warning to remix, then open an issue to update  and remove the deprecated tokens and replace them with the new ones?